bot/exts/evergreen/githubinfo.py

```diff
@@ -72,26 +76,93 @@ async def get_github_info(self, ctx: commands.Context, username: Optional[str])
 
             if user_data['type'] == "User":
 
-                embed.add_field(name="Followers",
-                                value=f"[{user_data['followers']}]({user_data['html_url']}?tab=followers)")
-                embed.add_field(name="\u200b", value="\u200b")
-                embed.add_field(name="Following",
-                                value=f"[{user_data['following']}]({user_data['html_url']}?tab=following)")
-
-            embed.add_field(name="Public repos",
-                            value=f"[{user_data['public_repos']}]({user_data['html_url']}?tab=repositories)")
-            embed.add_field(name="\u200b", value="\u200b")
+                embed.add_field(
+                    name="Followers",
+                    value=f"[{user_data['followers']}]({user_data['html_url']}?tab=followers)"
+                )
+                embed.add_field(
+                    name="Following",
+                    value=f"[{user_data['following']}]({user_data['html_url']}?tab=following)"
+                )
+
+            embed.add_field(
+                name="Public repos",
+                value=f"[{user_data['public_repos']}]({user_data['html_url']}?tab=repositories)"
+            )
 
             if user_data['type'] == "User":
-                embed.add_field(name="Gists", value=f"[{gists}](https://gist.github.com/{username})")
+                embed.add_field(name="Gists", value=f"[{gists}](https://gist.github.com/{quote(username, safe='')})")
 
-                embed.add_field(name=f"Organization{'s' if len(orgs)!=1 else ''}",
-                                value=orgs_to_add if orgs else "No organizations")
-                embed.add_field(name="\u200b", value="\u200b")
+                embed.add_field(
+                    name=f"Organization{'s' if len(orgs)!=1 else ''}",
+                    value=orgs_to_add if orgs else "No organizations"
+                )
             embed.add_field(name="Website", value=blog)
 
         await ctx.send(embed=embed)
 
+    @github_group.command(name='repository', aliases=('repo',))
+    async def github_repo_info(self, ctx: commands.Context, repo: str) -> None:
+        """Fetches a repositories' GitHub information. The repository should look like `user/reponame`."""
+        if repo.count('/') != 1:
+            embed = discord.Embed(
+                title=random.choice(NEGATIVE_REPLIES),
+                description="The repository should look like `user/reponame`.",
+                colour=Colours.soft_red
+            )
+
+            await ctx.send(embed=embed)
+            return
+
+        async with ctx.typing():
+            repo_data = await self.fetch_data(f"{GITHUB_API_URL}/repos/{quote(repo)}")
+
+            # There won't be a message key if this repo exists
+            if repo_data.get('message') is not None:
```

Comment:
Could be simplified to:
```suggestion
            if "message" in repo_data:
```